docs/releases/2.1.txt

```diff
@@ -183,7 +183,10 @@ Tests
 URLs
 ~~~~
 
-* ...
+* Added a system check warning if an URL pattern passed to
```

Comment:
No need for a release note -- anyway, I think this would be a candidate for 2.0.